### PR TITLE
Add the layout for Login and Admin page

### PR DESCRIPTION
**Pull request by ricardoruwer**
### Login page
localhost_4000_login

### Admin page
localhost_4000_admin

### CODE REVIEW

File: lib/construtora_lc_hiert/authentication/error_handler.ex [hunk 1]
```diff
@@ -7,10 +7,17 @@ defmodule ConstrutoraLcHiert.Authentication.ErrorHandler do
...
   end
+
+  def get_message(reason_type) do
+    case reason_type do
+      :unauthenticated -> "Não autenticado! Faça login."
+      :not_found -> "Usuário não encontrado."
+      :invalid_credentials -> "A senha está incorreta."
+      _ -> "Ocorreu um erro! Tente novamente."
+    end
+  end
```
Comment (ricardoruwer) [inline lib/construtora_lc_hiert/authentication/error_handler.ex]: What is the best way to translate these messages?

File: lib/construtora_lc_hiert_web/controllers/session_controller.ex [hunk 2]
```diff
@@ -30,14 +30,14 @@ defmodule ConstrutoraLcHiertWeb.SessionController do
...
-    |> put_flash(:success, "Welcome back!")
+    |> put_flash(:success, "Bem-vindo")
     |> Guardian.Plug.sign_in(user)
     |> redirect(to: Routes.admin_page_path(conn, :index))
   end
 
   defp login_reply({:error, reason}, conn) do
     conn
-    |> put_flash(:error, to_string(reason))
+    |> put_flash(:error, ErrorHandler.get_message(reason))
```
Comment (ricardoruwer) [inline lib/construtora_lc_hiert_web/controllers/session_controller.ex]: Is it ok to call the `Guardian.ErrorHandler` here inside the `sessions_controller` just to translate the messages?

File: lib/construtora_lc_hiert_web/templates/layout/admin.html.eex [hunk 3]
```diff
@@ -1,30 +1,63 @@
-<!DOCTYPE html>
-<html lang="en">
+<!doctype html>
+<html lang="pt-br">
```
Comment (ricardoruwer) [inline lib/construtora_lc_hiert_web/templates/layout/admin.html.eex]: The correct is `pt_BR` or `pt-br` or both or none?

File: lib/construtora_lc_hiert/authentication/error_handler.ex (hunk 1 above)
Comment (andrielfn) [reply lib/construtora_lc_hiert/authentication/error_handler.ex]: Gettext?

Comment (w-sanches) [reply lib/construtora_lc_hiert/authentication/error_handler.ex]: Translation as in portuguese to another language or `reason_type` to error message?

File: lib/construtora_lc_hiert_web/controllers/session_controller.ex (hunk 2 above)
Comment (w-sanches) [reply lib/construtora_lc_hiert_web/controllers/session_controller.ex]: I think it is okay to have an `ErrorHandler.get_message/1` function to handle the atom to string conversion, but this specific `ErrorHandler` module is bound to the authentication context. We could probably move this function to another module with the sole responsibility of converting atoms to messages, WDYT?

File: lib/construtora_lc_hiert/authentication/error_handler.ex (hunk 1 above)
Comment (ricardoruwer) [reply lib/construtora_lc_hiert/authentication/error_handler.ex]: `reason_type` to an error message.

File: lib/construtora_lc_hiert_web/templates/layout/admin.html.eex (hunk 3 above)
Comment (w-sanches) [reply lib/construtora_lc_hiert_web/templates/layout/admin.html.eex]: Checkout [this nice website](https://r12a.github.io/app-subtags/) for validating your language tag.

TL;DR: `pt-br` is the correct one, but you could write it like `pt-BR` to separate the language from the region identifier.

File: lib/construtora_lc_hiert_web/controllers/session_controller.ex (hunk 2 above)
Comment (ricardoruwer) [reply lib/construtora_lc_hiert_web/controllers/session_controller.ex]: Hum... 🤔 
But these messages from `ErrorHandler.get_message/1` are specific for authentication as well, right?

```elixir
:unauthenticated -> "Não autenticado! Faça login."
:not_found -> "Usuário não encontrado."
:invalid_credentials -> "A senha está incorreta."
_ -> "Ocorreu um erro! Tente novamente."
```

Comment (w-sanches) [reply lib/construtora_lc_hiert_web/controllers/session_controller.ex]: I guess I am still half asleep 😴 
Everything looks fine by me then :D

You are not calling `Guardian.ErrorHandler`, but your own error handler module

File: lib/construtora_lc_hiert/authentication/error_handler.ex (hunk 1 above)
Comment (w-sanches) [reply lib/construtora_lc_hiert/authentication/error_handler.ex]: I wouldn't do it any different, maybe use pattern matching as multiple if conditions 🙈 

```Elixir
def get_message(:unauthenticated), do: "Não autenticado! Faça login."
def get_message(:not_found), do: "Usuário não encontrado."
def get_message(:invalid_credentials), do: "A senha está incorreta."
def get_message(_), do: "Ocorreu um erro! Tente novamente."
```

This looks just a tiny little bit better to me...
